### ISSUE TITLE
Return "objects" instead of "docs"

### ISSUE THREAD

**Issue by CardboardMechanic** [LeakyBucket/google_apps#8]
```
1.9.3-p385 :004 > client.get_users.first
 => <?xml version="1.0" encoding="UTF-8"?>
<atom:entry xmlns:atom="http://www.w3.org/2005/Atom" xmlns:apps="http://schemas.google.com/apps/2006" xmlns:gd="http://schemas.google.com/g/2005">
<atom:category scheme="http://schemas.google.com/g/2005#kind" term="http://schemas.google.com/apps/2006#user"/>
  <apps:login userName="will" suspended="false" ipWhitelisted="false" admin="false" changePasswordAtNextLogin="true" agreedToTerms="true"/>
  <apps:quota limit="25600"/>
  <apps:name familyName="Read" givenName="Will"/>
  <gd:feedLink rel="http://schemas.google.com/apps/2006#user.nicknames" href="https://apps-apis.google.com/a/feeds/loopb.ac/nickname/2.0?username=will"/>
  <gd:feedLink rel="http://schemas.google.com/apps/2006#user.emailLists" href="https://apps-apis.google.com/a/feeds/loopb.ac/emailList/2.0?recipient=will%40loopb.ac"/>
</atom:entry>
```

I feel like I want this to return something more traditional in terms of ruby objects so that the accessible attributes are more discoverable. I like that there's an XML document backing the object so that it can be posted at any moment, but that feels more like a property rather than the primary representation since the point of a library is to abstract away the implementation of the Google API. Thoughts?

**Comment by LeakyBucket** [comment]
I agree completely. That was the idea with Atom#add_doc_dispatcher and the optional xml argument in the document initalizers.  The issue I ran into was finding a good medium for the opposite natures of the two.  I was constantly fighting with the fact that the essence of XML is structure.  I threw the MAP hashes into the Atom Document Classes as a "solution" to presenting only what really mattered while being able to translate it back to what XML cared about.

Maybe a better solution is another class that sits in-between and performs those behaviors?

That being said, the above behavior might be a bug.  As long as get_users is returning a Feed then first should be a User instance.
